### DISCUSSION TITLE
GitOpsCon EU 2023 - Virtual Event (2days or 1 day?)

### DISCUSSION THREAD

**Discussion by christianh814** [open-gitops/project#167] [Polls]
The LinuxFoundation is going to do GitOpsCon EU 2023 as a virtual event and is giving us Dec 5th and/or 6ths for the Event CFP opens August 15th. We discussed during the call if we should do 1 full day event or 2 half day events

**Comment by christianh814** [comment]
I'm thinking 2 half days in the afternoon ([1300 CET - 1700 CET](https://everytimezone.com/s/69de7bae)) would be easier to attend (small, consumable, pieces). Also doing it this way we'll be able to catch the east coast US and parts of APAC.

**Reply by sourabhdeshmukh** [reply]
Yes. That way more audience will able to attend the event.

**Reply by scottrigby** [reply]
I like this too. Since it's virtual there's little benefit in making it a single marathon event. Good thinking!

**Comment by staceypotter** [comment]
Can we close this please?
